spec/models/spotlight/blacklight_configuration_spec.rb

```diff
@@ -450,4 +450,40 @@
       end
     end
   end
+
+  describe 'document_index_view_types=' do
+    it 'Array should pass through' do
```

Comment:
Can we rephrase this to: `passes arrays through` (and below, `is simplified from a hash to an array`)?